bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
It looks like files in this `bin` directory are added to the gem as executables.

https://github.com/envato/rack-ecg/blob/b6f21a45fba86b82897577fa49517b1868d339ac/rack-ecg.gemspec#L22

I don't think we should include these as they're focused on helping in the development of the gem.

Reply:
I was trying to remember why we didn't have binstubs included, that would be why. 😅

Reply:
e276a34e7c8c9c7d18fd2639bdea12c5ce3015bc includes a shift to using `exe` as the `bindir`, among other changes pulled from the gemspec of a new `bundle gem`.